### PR TITLE
Refine metrics

### PR DESCRIPTION
**Pull request by kciesielski**
Metrics after this update:
1. ~sender throughput~ removed
2. ~sender latency~ removed
3. receiver throughput 
4. receiver latency (time to read a single batch)
5. cluster latency (based on timestamp in message)

### CODE REVIEW

File: src/main/scala/com/softwaremill/mqperf/mq/Mq.scala [hunk 1]
```diff
@@ -12,6 +12,10 @@ trait Mq {
     def send(msgs: List[String])
 
     def close() {}
+
+    def withTimestamp(msg: String): String = msg + "_" + System.currentTimeMillis().toString
```
Comment (mkubala) [inline src/main/scala/com/softwaremill/mqperf/mq/Mq.scala:16]: How about cutting off the last N characters from msg, where N is the number of digits in the timestamp?
This way we will preserve the message length described in `msg_size` test parameter.

File: src/main/scala/com/softwaremill/mqperf/Receiver.scala [hunk 2]
```diff
@@ -52,26 +56,32 @@ class ReceiverRunnable(
...
-  private def doReceive(mqReceiver: mq.MqReceiver, msgTimer: Timer) = {
+  private def doReceive(mqReceiver: mq.MqReceiver, msgTimer: Timer, clusterTimer: Timer) = {
     val before = System.nanoTime()
     val msgs = mqReceiver.receive(receiveMsgBatchSize)
     if (msgs.nonEmpty) {
       val after = System.nanoTime()
+      val afterMs = after / 1000000L
+      msgs.foreach {
+        case (_, msg) =>
+          val msgTimestamp = msg.substring(msg.indexOf('_') + 1).toLong
```
Comment (mkubala) [inline src/main/scala/com/softwaremill/mqperf/Receiver.scala:82]: We could move appropriate helper method from `Mq.Sender` to `Mq` and reuse it here.

**Review by mkubala — APPROVED** [review]
I left some comments but those are rather minors.

File: src/main/scala/com/softwaremill/mqperf/mq/Mq.scala (hunk 1 above)
Comment (kciesielski) [reply src/main/scala/com/softwaremill/mqperf/mq/Mq.scala:16]: Good point, I'll update this.

File: src/main/scala/com/softwaremill/mqperf/Receiver.scala (hunk 2 above)
Comment (kciesielski) [reply src/main/scala/com/softwaremill/mqperf/Receiver.scala:82]: Maybe I'll just change the way timestamp is serialized into message: msg = timestamp (as string) prepended with zeroes to fill up to required length in bytes. This way the "deserialization" would be just a `.toLong` call.

Comment (mkubala) [reply src/main/scala/com/softwaremill/mqperf/Receiver.scala:82]: I ❤️ the idea with leading zeroes.

**Pull request merged**